Officially extend the call for venue proposals (#76)
DEFNA/defna.org: _posts/2024-10-16-djangocon-us-call-for-venue-proposal-2025-extension.md

---
layout: post
title:  "DjangoCon US: Call for Venue Proposal 2025 extension"
date: 2024-10-16 12:00:00
categories: ""
author_name : DEFNA
author_url : /about
author_avatar: defna
show_avatar : true
read_time : ""
feature_image: 2016-06-08
show_related_posts: false
permalink: /announcements/2024/10/16/djangocon-us-call-for-venue-proposal-2025-extension/
---

First off, thank you to the teams who sent in proposals. We're very appreciate of the time and effort that you've put in to this point.

However, since we have yet to find a proposal that meets all of our needs we're officially re-opening our call of venue proposals.

If you are interested in bringing DjangoCon US to your city for 2025 and/or 2026, please submit your proposal to [hello@defna.org](mailto:hello@defna.org) no later than November 15, 2024.

> For 2025, we are looking at conference dates of October 5-10, 2025 or October 12-17, 2025.

Reviewer: The DSF got upset with us for conflicting with Sukkot in 2022 after they signed off on it. I'd like to avoid the latter week if possible to dodge that a second time.

> For 2025, we are looking at conference dates of October 5-10, 2025 or October 12-17, 2025.

Author: ```suggestion
For 2025, we are looking at conference dates of October 5-10, 2025. We are no longer interested in dates of October 12-17, 2025 as they conflict with Sukkot.
```

> For 2025, we are looking at conference dates of October 5-10, 2025. We are no longer interested in dates of October 12-17, 2025 as they conflict with Sukkot.

Author: ```suggestion
For 2025, we are looking at conference dates during the following times. Please note, some may be preferred more than others. For example, if possible we'd like to avoid October 12-17 as they conflict with Sukkot.

- August 31-September 5, 2025
- September 7-12, 2025
- September 14-19, 2025
- September 21-26, 2025
- October 5-10, 2025
- October 12-17, 2025
- October 19-24, 2025
- October 26-31, 2025
```

---
layout: post
title:  "DjangoCon US: Call for Venue Proposal 2025 extension"
date: 2024-10-16 12:00:00
categories: ""
author_name : DEFNA
author_url : /about
author_avatar: defna
show_avatar : true
read_time : ""
feature_image: 2016-06-08
show_related_posts: false
permalink: /announcements/2024/10/16/djangocon-us-call-for-venue-proposal-2025-extension/
---

First off, thank you to the teams who sent in proposals. We're very appreciate of the time and effort that you've put in to this point.

However, since we have yet to find a proposal that meets all of our needs we're officially re-opening our call of venue proposals.

If you are interested in bringing DjangoCon US to your city for 2025 and/or 2026, please submit your proposal to [hello@defna.org](mailto:hello@defna.org) no later than November 15, 2024.

For 2025, we are looking at conference dates during the following times. Please note, some may be preferred more than others. For example, if possible we'd like to avoid October 12-17 as they conflict with Sukkot.

- August 31-September 5, 2025
- September 7-12, 2025
- September 14-19, 2025

> - September 21-26, 2025

Reviewer 2: ```suggestion
```
I'm seeing a conflict on this one [Rosh Hashanah](https://www.chabad.org/library/article_cdo/aid/4644/jewish/Rosh-Hashanah-2024.htm)
Begins sunset of  Monday, September 22, 2025
Ends nightfall of  Wednesday, September 24, 2025

Author: In our slack thread, we talked about including it with the (internal) caveat it may be excluded later because of the holiday.
